### PR TITLE
configs: Add j784s4 configuration

### PR DESCRIPTION
**Pull request by 3V3RYONE**
Introduces `j784s4_evm_gp_all_defconfig` and `j784s4_evm_hs_fs_all_defconfig` configs to add support for building j784s4-evm and AM69-SK platforms. 

How was this tested?
- Build and Boot j784s4-evm (gp)
- Build and Boot AM69-SK (hs-fs)

### CODE REVIEW

File: configs/j784s4_evm_gp_all_defconfig [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# J784s4 EVM configuration for GP devices
```
Comment (nmenon) [inline configs/j784s4_evm_gp_all_defconfig:1]: NO. Delete this - GP should never exist.

File: configs/j784s4_evm_hs_fs_all_defconfig [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# J784s4 EVM configuration for HS-FS devices
```
Comment (nmenon) [inline configs/j784s4_evm_hs_fs_all_defconfig]: Why are you deleting the configuration file and then introducing this file again. just introduce the AM69 SK file as a new file.

**Review by nmenon — CHANGES_REQUESTED** [review]

File: configs/j784s4_evm_gp_all_defconfig (hunk 1 above)
Comment (3V3RYONE) [reply configs/j784s4_evm_gp_all_defconfig:1]: Sorry, but requesting a bit of clarification. We do have GP configuration for [J721S2-EVM](https://github.com/nmenon/k3-upstream-boot-build/blob/main/configs/j721s2_evm_gp_all_defconfig#L1) and [J721E-EVM](https://github.com/nmenon/k3-upstream-boot-build/blob/main/configs/j721e_evm_gp_all_defconfig#L1) in the repo. Are we not trying to do the same for J784S4-EVM? I could be totally wrong, but we do test J784S4 GP EVM configurations in the test-farm internally, right?  

Please feel free to address my queries whenever you have time. Thanks :slightly_smiling_face:

File: configs/j784s4_evm_hs_fs_all_defconfig (hunk 2 above)
Comment (3V3RYONE) [reply configs/j784s4_evm_hs_fs_all_defconfig]: Got it, will introduce a new file.. Thanks :slightly_smiling_face:

File: configs/j784s4_evm_gp_all_defconfig (hunk 1 above)
Comment (glneo) [reply configs/j784s4_evm_gp_all_defconfig:1]: J721e came out before we started going all in on HS devices, so they get a pass (for now). All new devices should be HS only from the start.

**Comment by 3V3RYONE** [comment]
Closing this as #19 got merged

**Pull request closed without merging**